docs/CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ Thank you for your interest in contributing to JsDbg! Code contributations, as w
 ### Getting the source
 The JsDbg source can be obtained from the [JsDbg repository on GitHub](https://aka.ms/jsdbg). The URL for cloning the repository is: `https://github.com/MicrosoftEdge/JsDbg.git`.
 
-### Building JsDbg
+### Building JsDbg (Windows)
```

Comment:
Instead of "Building JsDbg (Windows)" and "Building JsDbg (Linux)", can we keep "Building JsDbg" as is and add subheadings for 'Windows" and "Linux"?

Reply:
Done